### PR TITLE
fix: [CDS-70851]: warning message on duplicate job parameters

### PR DESCRIPTION
**Pull request by satyam-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by kalyansai-harness** [comment]
@satyam-harness there are so many checks failing here. Could you please check this.

**Comment by satyam-harness** [comment]
retrigger prettier

**Comment by satyam-harness** [comment]
retrigger titlecheck

**Review by litvinovstas — DISMISSED** [review]

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepBase.tsx [hunk 1]
```diff
@@ -532,101 +544,124 @@ function FormContent({
...
+                icon="warning-sign"
+                intent="warning"
+                iconProps={{ size: 18, color: Color.RED_800, padding: { right: 'small' } }}
+                color={Color.RED_700}
+                font={{ weight: 'semi-bold' }}
+              >
+                {getString('pipeline.jenkinsStep.jobParameterDuplicateWarning')}
+              </Text>
+              <Button
+                aria-label="hide warning"
```
Comment (rameshx) [inline src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepBase.tsx:560]: Add to strings

**Review by rameshx — DISMISSED** [review]

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepBase.tsx (hunk 1 above)
Comment (satyam-harness) [reply src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepBase.tsx:560]: sure

**Review by rameshx — APPROVED** [review]

**Review by litvinovstas — APPROVED** [review]

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepBase.tsx (hunk 1 above)
Comment (satyam-harness) [reply src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepBase.tsx]: done

**Pull request merged**